Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -168,6 +175,11 @@ public void tankDrivePowerMode(double leftPower, double rightPower) {
         // Coerce powers into appropriate limits
         leftPower = MathUtils.constrainDoubleToRobotScale(leftPower);
         rightPower = MathUtils.constrainDoubleToRobotScale(rightPower);
+        
+        if(precisionMode.get()){
+            leftPower/=precisionFactor.get();
```

Comment:
At least to me, this would feel more natural as multiplication. It's much easier to judge values without the inverse proportion.